src/utils/emails.ts

```diff
@@ -1,28 +1,43 @@
 import nodemailer from 'nodemailer';
+import dotenv from 'dotenv';
+dotenv.config();
 
 const { NODEMAILER_EMAIL_USERNAME, NODEMAILER_EMAIL_PASSWORD, PORT } =
   process.env;
 
 // CONFIGURE HOST
 const host: string =
   process.env.NODE_ENV === 'production'
-    ? process.env.HOST!
+    ? process.env.API_URL!
     : `http://localhost:${PORT}`;
 
-const registerMessageTemplate = (firstName: string, token: string): string => `
-Dear ${firstName},
-You have successfully registered on E-COMMERCE-ONES-AND-ZEROES. We are happy to have you on board.
+const registerMessageTemplate = (firstName: string, token: string): any => `
+
+<div style="font-family: Arial, sans-serif; color: #333; line-height: 1.6; background-color: #fff; margin: 0; padding: 0;">
+    <div style="max-width: 600px; margin: 20px auto; padding: 20px; border: 1px solid #ddd; border-radius: 10px; background-color: #ffffff;">
+        <div style="font-size: 18px; font-weight: bold; margin-bottom: 20px;">Dear ${firstName},</div>
+        <p style="font-size: 16px; font-weight: bold;">You have successfully registered on E-COMMERCE-ONES-AND-ZEROES. We are happy to have you on board.</p>
+        <p style="font-size: 16px;">Please click on the following button to verify your account:</p>
+        
+        <div style="text-align: center;">
+            <a href="${host}/api/users/isVerified/${token}" style="text-decoration: none;">
```

Comment:
Good work celestin! but instead of sending link to the user with backend routes why cant we send frontend link with token attached to it so that if the user clicks verify button he/ she can be redirected to one of our frontend page and i think that is when we can be able to verify him using token in url params.

Reply:
Because we are verifying user up on link within email, we have to attach backend URL for verifying user so that we may automatically verify user only with clicking button within email  , we have not involve many steps on fronted as it may be considered as bad user experience.